### ISSUE TITLE
[BUG] Can't connect to the VM via SSH during initial setup

### ISSUE THREAD

**Issue by cid404** [code-ready/crc#2834]
During the `crc setup` a connection cannot be established via SSH to the VM.  Also CRC domains are not pingable/resolvable.

### General information

  * OS: Windows
  * Hypervisor:  Hyper-V 
  * Did you run `crc setup` before starting it? Yes
  * Running CRC on:  Desktop

## CRC version
```bash
# Put `crc version` output here

CodeReady Containers version: 1.35.0+751824a9
OpenShift version: 4.9.5 (bundle installed at C:\Program Files\CodeReady Containers\crc_hyperv_4.9.5.crcbundle)

```
  
## CRC status
```bash
# Put `crc status --log-level debug` output here

C:\Users\user1>crc status --log-level debug
DEBU CodeReady Containers version: 1.35.0+751824a9
DEBU OpenShift version: 4.9.5 (bundle installed at C:\Program Files\CodeReady Containers\crc_hyperv_4.9.5.crcbundle)
DEBU Running 'crc status'
DEBU Checking file: C:\Users\user1\.crc\machines\crc\.crc-exist
DEBU Checking file: C:\Users\user1\.crc\machines\crc\.crc-exist
DEBU Running 'Hyper-V\Get-VM crc | Select-Object -ExpandProperty State'
DEBU Running SSH command: df -B1 --output=size,used,target /sysroot | tail -1
DEBU Using ssh private keys: [C:\Users\user1\.crc\machines\crc\id_ecdsa C:\Users\user1\.crc\cache\crc_hyperv_4.9.5\id_ecdsa_crc]
DEBU SSH command results: err: ssh: handshake failed: read tcp 127.0.0.1:64939->127.0.0.1:2222: wsarecv: An existing connection was forcibly closed by the remote host., output:
DEBU Cannot get root partition usage: ssh command error:
command : df -B1 --output=size,used,target /sysroot | tail -1
err     : ssh: handshake failed: read tcp 127.0.0.1:64939->127.0.0.1:2222: wsarecv: An existing connection was forcibly closed by the remote host.
DEBU cannot get OpenShift status: CreateFile C:\Users\user1\.crc\machines\crc\kubeconfig: The system cannot find the file specified.
CRC VM:          Running
OpenShift:       Unreachable (v4.9.5)
Disk Usage:      0B of 0B (Inside the CRC VM)
Cache Usage:     14.94GB
Cache Directory: C:\Users\user1\.crc\cache


```

## CRC config
```bash
# Put `crc config view` output here

crc config view
- consent-telemetry                     : no
```

## Host Operating System
```bash
# Put the output of `cat /etc/os-release` in case of Linux
# put the output of `sw_vers` in case of Mac
# Put the output of `systeminfo` in case of Windows


Host Name:                 HOME
OS Name:                   Microsoft Windows 10 Pro
OS Version:                10.0.19043 N/A Build 19043
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Standalone Workstation
OS Build Type:             Multiprocessor Free
Registered Organization:
System Boot Time:          10-Nov-21, 5:31:40 PM
System Manufacturer:       Micro-Star International Co., Ltd.
System Model:              MS-7C37
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                           [01]: AMD64 Family 23 Model 113 Stepping 0 AuthenticAMD ~3800 Mhz
BIOS Version:              American Megatrends Inc. H.00, 04-Jun-19
Windows Directory:         C:\WINDOWS
System Directory:          C:\WINDOWS\system32
Boot Device:               \Device\HarddiskVolume2
System Locale:             en-us;English (United States)
Total Physical Memory:     32,717 MB
Available Physical Memory: 27,279 MB
Virtual Memory: Max Size:  37,581 MB
Virtual Memory: Available: 29,603 MB
Virtual Memory: In Use:    7,978 MB
Page File Location(s):     C:\pagefile.sys
Domain:                    WORKGROUP
Logon Server:              \\HOME
Network Card(s):           5 NIC(s) Installed.
                           [01]: VMware Virtual Ethernet Adapter for VMnet1
                                 Connection Name: VMware Network Adapter VMnet1
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 192.168.189.1
                                 [02]: fe80::a50e:f5a6:7fc1:df62
                           [02]: VMware Virtual Ethernet Adapter for VMnet8
                                 Connection Name: VMware Network Adapter VMnet8
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 192.168.83.1
                                 [02]: fe80::784f:cea5:acbe:3e76
                           [03]: Realtek PCIe GbE Family Controller
                                 Connection Name: Ethernet
                                 DHCP Enabled:    Yes
                                 DHCP Server:     192.168.1.1
                                 IP address(es)
                                 [01]: 192.168.1.22
                                 [02]: fe80::3073:9af6:d507:f64f
                           [04]: Bluetooth Device (Personal Area Network)
                                 Connection Name: Bluetooth Network Connection
                                 Status:          Media disconnected
                           [05]: Hyper-V Virtual Ethernet Adapter
                                 Connection Name: vEthernet (Default Switch)
                                 DHCP Enabled:    No
                                 IP address(es)
                                 [01]: 172.29.16.1
                                 [02]: fe80::d5e5:3470:f62f:ecb7
Hyper-V Requirements:      A hypervisor has been detected. Features required for Hyper-V will not be displayed.


```

### Steps to reproduce

  1. crc setup --log-level debug
  2. crc start--log-level debug
 
### Expected

Setup completes successfully.

### Actual

Setup fails midway

### Logs

Before gather the logs try following if that fix your issue
```bash
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug
```

https://gist.github.com/cid404/aeb2ae79b2406873db4b7ef00b8aecb0

**Comment by cuongtsm** [comment]
I'm still getting this probem after run all the comment above

**Comment by cfergeau** [comment]
@cuongtsm it's better if you open a new issue and follow the issue template to provide the information that we need to assist you.

**Comment by gbraad** [comment]
the OP had VMware and Hyper-V installed. This needs modification with `bcdedit` to work properly. That is not a supported setup.

Please file a new issue with your setup info.
